src/app/loginComponents/auth/auth.component.ts

```diff
@@ -51,10 +51,14 @@ export class AuthComponent implements OnInit, OnDestroy {
             return addGitHubToken;
           case 'quay':
             return addQuayToken;
```

Comment:
Please add some comments here. 
This seems oddly internally inconsistent (github, not github.com vs gitlab.com) and doesn't seem to quite match the webservice either https://github.com/ga4gh/dockstore/blob/develop/dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java#L163 ((quay, vs quay.io). 

We don't have to fix it across the board for this release, but leave good notes for the next developer to understand/clean this up.